src/plugins/bfetch/public/batching/create_streaming_batched_function.ts

```diff
@@ -134,6 +135,11 @@ export const createStreamingBatchedFunction = <Payload, Result extends object>(
               if (response.error) {
                 items[response.id].future.reject(response.error);
               } else if (response.result !== undefined) {
+                if (typeof response.result === 'object') {
+                  response.result.meta = {
```

Comment:
i don't really like it that we are adding things to actual response, i would preffer to se meta information separate from actual responses.

also, i think this information could be useful on the server as well, and i would really prefer to keep our server/client API as similar as possible.